src/main/java/my/test/example/LoginPage.java

```diff
@@ -0,0 +1,51 @@
+package my.test.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage {
+
+    WebDriver driver = new ChromeDriver();
+
+    public void login(String login, String password) {
```

Comment:
Optional (out of task's scope): Page Object pattern task will be in task 60. According to classic PO pattern every public method must return current page/new page or some value (String, boolean, any), but not void.